presenter/gitiles/gitiles_test.go

```diff
@@ -22,7 +23,7 @@ func TestFetchLog(t *testing.T) {
 		}),
 	}
 
-	log, err := fetchLog(client, "")
+	log, err := fetchLog(context.Background(), client, "")
```

Comment:
Shouldn't that be `context.TODO()` since it's not really something intended to run in the background? I ask because I really don't know 😄